File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewMyListOfImagesRow.cpp

#include "ListViews_ViewMyListOfImagesRow.h"
#include "ListViews_ViewMyListOfImagesRow.h"
#include <string>

namespace widgetassertions
{
  std::string ListViews_ViewMyListOfImagesRow::getMyInfoImageImageSource( ) 
  {
    return this->MyInfoImageImageSource;
  }
  bool ListViews_ViewMyListOfImagesRow::getIsMyInfoImageEnabled( ) 
  {
    return this->isMyInfoImageEnabled;
  }
  bool ListViews_ViewMyListOfImagesRow::getIsMyInfoImageVisibility( ) 
  {
    return this->isMyInfoImageVisibility;
  }
  int ListViews_ViewMyListOfImagesRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/ListViews_ViewMyListOfImagesRow.cs
/// <filename>
///     ListViews_ViewMyListOfImagesRow.cs
/// </filename>


namespace widgetassertions
{
    internal class ListViews_ViewMyListOfImagesRow
  {
        private string MyInfoImageName;
     
        public     virtual string getMyInfoImageName()
    {
      return this.MyInfoImageName;
    }
     
        private bool isMyInfoImageEnabled;
     
        public     virtual bool getIsMyInfoImageEnabled()
    {
      return this.isMyInfoImageEnabled;
    }
     
        private bool isMyInfoImageVisible;
     
        public     virtual bool getIsMyInfoImageVisible()
    {
      return this.isMyInfoImageVisible;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

